lib/fn/index.js

```diff
@@ -22,12 +20,11 @@ export function normalizeParams(options={}) {
  * @param {(string|constructor)} [ioOpts.io] - If passing an object, a string specifying a path to the IO provider or a constructor
  * @ignore
  */
-export function normalizeIO(options={}) {
-  if (typeof options === "number" || typeof options === "string") {
-    let result = {pin: options};
-    return result;
+export function normalizeIO(pin={}) {
+  if (typeof pin === "number" || typeof pin === "string") {
+    return {pin};
   }
-  return options;
+  return pin;
```

Comment:
This is cleaner, but this last line gives me pause. If it falls through to here, we aren't necessarily returning a pin. It could be a more complex IO options object. It works of-course,  I'm just obsessing over the label.

Reply:
I spent several minutes staring at this one as well, mulling the same question you're asking. I ended up landing on liking the property shorthand more 🤷 my feelings wouldn't be hurt if you changed it back